packages/lib/services/plugins/PluginService.ts

```diff
@@ -416,7 +416,7 @@ export default class PluginService extends BaseService {
 		return this.installPluginFromRepo(repoApi, pluginId);
 	}
 
-	public async installPlugin(jplPath: string): Promise<Plugin> {
+	public async installPlugin(jplPath: string, defaultPlugin: boolean = false): Promise<Plugin> {
```

Comment:
I'd prefer we keep this method generic, with no mention of default plugins. Why is this option needed?

Reply:
If `defaultPlugin` is true, then we won't be loading the plugins right away after installing them. As we are installing default plugins before loading regular plugins, we just need to load them once along with regular plugins.

Reply:
Ok then please name the option as `loadPlugin`, defaults to `true`

Reply:
Done.